### PR TITLE
feat: apartment payment setting card implemwnted

### PR DESCRIPTION
**Pull request by nuelkaiser**
Screenshot (22)

### CODE REVIEW

File: package.json
Comment (olajiredominic) [inline package.json:1]: Why did you add this package. I didn't see where it was used. Can you remove this package from the application

**Review by olajiredominic — CHANGES_REQUESTED** [review]

File: package.json
Comment (nuelkaiser) [reply package.json:1]: the package has been removed
